api/serverless.yml

```diff
@@ -86,6 +86,13 @@ functions:
                   path: ${self:custom.versions.v1}/publications/{id}
                   method: GET
                   cors: true
+    getPublicationSeed:
+        handler: src/components/publication/routes.getPubSeed
+        events:
+            - http:
+                path:  ${self:custom.versions.v1}/publication/seed
```

Comment:
Again, the naming feels a bit awkward. Maybe something like "/seed-data-publications" would be suitable for this